ColorPairCode_functions.cpp

```diff
@@ -0,0 +1,53 @@
+#include <iostream>
+#include "TeleComColorCoder.h"
+#include <assert.h>
+
+#define LAST_COLORCODE  25
+#define FIRST_COLORCODE 1
+
```

Comment:
This file mostly has tests, so should it be named TestColorPairCode.cpp? But it also has a print.. function.
Can the print...function be made testable?